Snippets/Misc/01-misc.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  01-misc.swift
+//
+//  Copyright (c) PubNub Inc.
+//  All rights reserved.
+//
+//  This source code is licensed under the license found in the
+//  LICENSE file in the root directory of this source tree.
+//
+
+// snippet.import
+import PubNubSDK
+import Foundation
+
+// snippet.end
+
+// snippet.pubnub
+// Initializes a PubNub object with the configuration
+let pubnub = PubNub(
+  configuration: PubNubConfiguration(
+    publishKey: "demo",
+    subscribeKey: "demo",
+    userId: "myUniqueUserId"
+  )
+)
+
+// snippet.end
+
+// snippet.encrypt-data
+func encryptDataExample() throws {
```

Comment:
In previous snippets tag has been placed inside of function and here it captures whole function - is it intentional?